checker/jdk/index/src/java/util/StringTokenizer.java

```diff
@@ -332,7 +333,7 @@ public boolean hasMoreTokens() {
      * @exception  NoSuchElementException  if there are no more tokens in this
      *               tokenizer's string.
      */
-    public String nextToken() {
+    public @MinLen(1) String nextToken() {
```

Comment:
Because `@MinLen` is a Value Checker annotation, this annotations should be in 
org/checkerframework/common/value/statically-executable.astub rather than the Index Checker's annotated jdk.